src/triggers/timesheetsUpdate.trigger

```diff
@@ -0,0 +1,10 @@
+trigger timesheetsUpdate on Timesheet__c (before insert, before update) {
+          for(Timesheet__c ts : Trigger.New) {
+              if(ts.Statuss__c =='NOT SUBMITTED'&& ts.Submitted__c){
+                  ts.Statuss__c ='SUBMITTED';
+              }
+              else if(ts.Statuss__c =='SUBMITTED'&& !ts.Submitted__c){
```

Comment:
Is there any place where it's required to change the Submitted__c flag if the status switches to SUBMITTED?